astropy_healpix/core.py

```diff
@@ -77,10 +84,92 @@ def level_to_nside(level):
     nside : int
         The number of pixels on the side of one of the 12 'top-level' HEALPix tiles.
     """
+    level = np.asarray(level, dtype=np.int64)
+
     _validate_level(level)
     return 2 ** level
 
 
+def nside_to_level(nside):
+    """
+    Find the HEALPix level for a given nside. (this is given by log2(nside))
+
+    This function is the inverse of `level_to_nside`.
+
+    Parameters
+    ----------
+    nside : int
+        The number of pixels on the side of one of the 12 'top-level' HEALPix tiles.
+        Must be a power of two.
+
+    Returns
+    -------
+    level : int
+        The level of the HEALPix cells
+    """
+    nside = np.asarray(nside, dtype=np.int64)
+
+    _validate_nside(nside)
+    return np.log2(nside)
+
+
+def uniq_to_levelipix(uniq):
```

Comment:
@astrofrog @lpsinger @bmatthieu3 - what about the name here?

I mean "levelipix" is not very readable, because it's non-obvious that it's two "words".
Is "uniq_to_level_ipix" better, or generally: is there a better name?